docs/match.md

```diff
@@ -0,0 +1,376 @@
+---
+id: match
+title: Match
+sidebar_label: Match
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::caution
+
+[Match expression is natively supported since PHP 8 version.](https://www.php.net/manual/en/control-structures.match.php) We recommend using that instead of the munus match. Munus match is a great option for PHP 7 support.
```

Comment:
here I am wondering, because it is actually true, but ... Munus gives a more functional approach to match, which native php 8 does not do ... although it does not know if it changes a lot ...